Core/WKBackForwardListExtension.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WKBackForwardListExtension.swift
+//  DuckDuckGo
+//
+//  Copyright © 2022 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import WebKit
+
+public extension WKBackForwardList {
+
+    func index(of backForwardListItem: WKBackForwardListItem) -> Int? {
```

Comment:
Naming nitpick: change `index(of:)` to `relativeOffset(of:)` ?

Reply:
added help message for it and renamed `var index` to `itemOffset` in the caller